static/js/colorspace.ts

```diff
@@ -31,27 +37,18 @@ export function luminance_to_lightness(luminance) {
     return 116 * v - 16;
 }
 
-export function getDecimalColor(hexcolor) {
+export function getDecimalColor(hexcolor: string): RGBColor {
     return {
         r: Number.parseInt(hexcolor.slice(1, 3), 16),
         g: Number.parseInt(hexcolor.slice(3, 5), 16),
         b: Number.parseInt(hexcolor.slice(5, 7), 16),
     };
 }
 
-export function getLighterColor(rgb, lightness) {
+export function getLighterColor(rgb: RGBColor, lightness: number): RGBColor {
     return {
         r: Math.round(lightness * 255 + (1 - lightness) * rgb.r),
         g: Math.round(lightness * 255 + (1 - lightness) * rgb.g),
         b: Math.round(lightness * 255 + (1 - lightness) * rgb.b),
     };
 }
```

Comment:
These are also unused.

Reply:
Updated to remove these methods as well.